content/resources/blogs/2021-12-16-theia-i18n/index.md

```diff
@@ -3,6 +3,8 @@ date = "2021-12-16"
 title = "Internationalization in Eclipse Theia"
 summary = "Even though some attempts by contributors have been made in the past, as of mid 2021 still Theia had no support for different locales. Arduino has addressed this with  the all new Arduino IDE based on Theia. Having supported Arduino previously with their IDE, they tasked TypeFox with contributing this feature to Theia."
 category = "blog"
+author = "Brian King"
+email = "briks.si@gmail.com"
```

Comment:
We should be careful attributing this blog to Brian, as the first byline states it was written by someone else

Reply:
Good catch! Will confirm with Clark

Reply:
Updated to Mark Sujew